chrome/content/zotfile/zotfile.js

```diff
@@ -455,16 +455,15 @@ Zotero.ZotFile = new function() {
      */
     this.formatSubfolder = function(item, format) {
         if (format == '') return '';
-        if(format.slice(-1) == this.folderSep) format = format.slice(0, -1);
+        if (format.slice(-1) == this.folderSep) format = format.slice(0, -1);
+        if (format[0] == this.folderSep) format = format.slice(1);
         var subfolder = this.Wildcards.replaceWildcard(item, format);
-        if (subfolder[0] == this.folderSep) subfolder = subfolder.slice(1);
```

Comment:
I changed the removing of heading `folderSep` before the wildcards being replaced.
Consider this format: `%a/%y`. When `authors` is available, e.g. `zhang`, we will get `zhang/2019`; when not, it should be `undefined/2019`.
however currently we'll have `/2019` after wildcards are replaced, and removing of heading `folderSep` will change it to `2019` instead of `undefined/2019`.